pkg/csi/service/vanilla/controller.go

```diff
@@ -282,7 +277,6 @@ func (c *controller) Init(config *cnsconfig.Config, version string) error {
 		return err
 	}
 
-	go cnsvolume.ClearTaskInfoObjects()
```

Comment:
Why was this removed? We still need this goroutine right?

Reply:
This goroutine cleans up the expired taskInfo objects from volumeTaskMap and snapshotTaskMap. This is still being called from the WCP controller, but in which case will it be required in Vanilla controller?

Reply:
This is a continuous cleanup process we perform to remove expired tasks from our cache. Without this, we will end up bloating our memory usage. If 1000 CreateVolume operations are called in the last 1 hour, we store the info of all these tasks in our cache, this needs to be cleared up at some point.

Reply:
My bad, I confused this go routine with https://github.com/kubernetes-sigs/vsphere-csi-driver/blob/master/pkg/internalapis/cnsvolumeoperationrequest/cnsvolumeoperationrequest.go#L122. I guess we can remove this, ignore my earlier comment.